SmilesMiles/View/TripView.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  TripView.swift
+//  SmilesMiles
+//
+//  Created by Pavan Sai Nallagoni on 2/8/23.
+//
+
+import SwiftUI
+
+struct TripView: View {
+    @State private var trips = ["New York"] //  [String]()
+    @State private var searchText = ""
+    @State private var isPresentingNewTrip = false
+    @State private var selectedTrip: String?
+    var body: some View {
+        NavigationView {
+            ZStack {
+                VStack {
+                    List {
+                        let filteredTrips = trips.filter { searchText.isEmpty || $0.localizedCaseInsensitiveContains(searchText) }
```

Comment:
Note that right now your code is not unit testable (unless you plan to unit-test your views).  We don't usually unit-test views because views, arguably, should be UI-tested (with automated UI tests, not automated unit tests).  Instead, we use _view models_.  You could put all the non-UI business-logic code in a view model and use the view model in your view and then unit-test the view model.